museum_export/process_one_museum_object.py

```diff
@@ -0,0 +1,106 @@
+# process_one_museum_object.py
+""" Module to do processing for one museum object """
+import os
+import time
+import json
+from clean_up_content import CleanUpContent
+from convert_json_to_csv import ConvertJsonToCsv
+from dependencies.pipelineutilities.validate_json import validate_json, get_nd_json_schema
+from dependencies.pipelineutilities.s3_helpers import write_s3_file
```

Comment:
I would recommend write_s3_json instead for writing json.
It does this:   
    s3.Object(s3Bucket, s3Path).put(Body=json.dumps(json_hash), ContentType='text/json')

Then when you try to open the file in the s3 console it will open it correctly in your browser.